package.json

```diff
@@ -69,13 +69,13 @@
     ]
   },
   "dependencies": {
-    "iterall": "^1.2.2",
-    "tslib": "^1.9.3"
+    "iterall": "^1.3.0",
+    "tslib": "^2.0.3"
```

Comment:
Note https://github.com/microsoft/tslib/releases/tag/2.0.0

Requires TS 3.9+

But should be fine for us here since https://github.com/graphql-nexus/schema/issues/580

Reply:
Prettier needs to be bumped to 2.2 if you want to use any of the new template string features FYI, not that needs to happen here since we're not using any yet (and should probably hold off unless there's a compelling need).

Reply:
Does tslib need to change to use 4.1?

Reply:
> Does tslib need to change to use 4.1?

Not that I know of, but I figured we should update it as a matter of housekeeping.